client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -126,3 +136,20 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
```

Comment:
There is still quite a bit of non-es6 syntax in this file, wasn't sure if you wanted to cleanup all the `var` usage in here while updating or grab it in a future PR.